# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/PDocumentChannel.ipdl
# Commit: 3998aeb0935c
# Full Hash: 3998aeb0935c84872b9eb63e74cc5563f8736f6b
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2020-06-12 14:56:55
# Regressor Bug: 1641737
# File Overlap Count: 1
# Description:
#   Bug 1641737 - P5. Remove unnecessary method. r=mattwoodrow
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78210
# ==============================================================================

diff -r f6c685e4b841 -r 3998aeb0935c netwerk/ipc/PDocumentChannel.ipdl
--- a/netwerk/ipc/PDocumentChannel.ipdl	Fri Jun 12 05:56:39 2020 +0000
+++ b/netwerk/ipc/PDocumentChannel.ipdl	Fri Jun 12 05:56:46 2020 +0000
@@ -52,11 +52,6 @@
   // and notifies the listener via a redirect to the new channel.
   async RedirectToRealChannel(RedirectToRealChannelArgs args, Endpoint<PStreamFilterParent>[] aEndpoint)
       returns (nsresult rv);
-
-  // Tell child to delete channel (all IPDL deletes must be done from child to
-  // avoid races: see bug 591708).
-  async DeleteSelf();
-
 };
 
 } // namespace net
